walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/comparator/SortEventsByNumAttendees.java

```diff
@@ -0,0 +1,96 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.comparator;
+
+import com.google.sps.Event;
+import com.google.sps.TimeRange;
+import com.google.sps.algorithms.BinarySearch;
+import com.google.sps.algorithms.MergeSort;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator; 
+import java.util.Set;
+
+/** 
+   * Used for sorting in descending order of the number optional attendees and 
+   *     for a tie breaker in descending order of duration.
+  */
+public class SortEventsByNumAttendees implements Comparator<Event> { 
+  private final String[] attendees;
+
+  public SortEventsByNumAttendees(Collection<String> attendees) {
+    ArrayList<String> attendeesList = Collections.list(Collections.enumeration
+        (attendees));
+
+    // order the array
+    MergeSort<String> merge = new MergeSort<String>();
+    merge.sort(attendeesList, String.CASE_INSENSITIVE_ORDER);
+
+    String[] attendeesArray = new String[attendeesList.size()];
+    attendeesArray = attendeesList.toArray(attendeesArray);    
+
+    this.attendees = attendeesArray;
```

Comment:
This can be simplified to one line:

```suggestion
    this.attendees = attendeesList.toArray(new String[attendeesList.size()]);
```